File: src/main/kotlin/io/github/dqualizer/dqlang/types/dam/architecture/apischema/data/DataSchema.java
package io.github.dqualizer.dqlang.types.dam.architecture.apischema.data;

import lombok.*;

import java.util.Set;

@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
@Data
public class DataSchema {

    private String type;
    private Set<Property> properties;
}

File: src/main/kotlin/io/github/dqualizer/dqlang/archive/openapiadapter/dqlang/data/DataSchema.kt
package io.github.dqualizer.dqlang.archive.openapiadapter.dqlang.data

import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.ToString

@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
class DataSchema {
    val type: String? = null
    val properties: LinkedHashSet<io.github.dqualizer.dqlang.archive.openapiadapter.dqlang.data.Property>? = null
}
